### PR TITLE
[FEAT] Dogecoin

### PR DESCRIPTION
**Pull request by gabrielbazan7**

### CODE REVIEW

File: src/providers/incoming-data/incoming-data.ts [hunk 1]
```diff
@@ -452,6 +464,21 @@ export class IncomingDataProvider {
     }
   }
 
+  private handleDogeUri(data: string, redirParams?: RedirParams): void {
+    this.logger.debug('Incoming-data: Doge URI');
+    let amountFromRedirParams =
+      redirParams && redirParams.amount ? redirParams.amount : '';
+    const coin = Coin.BTC;
```
Comment (Gamboster) [inline src/providers/incoming-data/incoming-data.ts:471]: Coin.BTC
should it be doge?

File: src/providers/incoming-data/incoming-data.ts [hunk 2]
```diff
@@ -579,6 +606,25 @@ export class IncomingDataProvider {
     }
   }
 
+  private handlePlainDogeAddress(
+    data: string,
+    redirParams?: RedirParams
+  ): void {
+    this.logger.debug('Incoming-data: Doge plain address');
+    const coin = Coin.BTC;
```
Comment (Gamboster) [inline src/providers/incoming-data/incoming-data.ts:614]: same here

File: src/providers/currency/coin.ts [hunk 3]
```diff
@@ -413,5 +413,42 @@ export const availableCoins: CoinsMap<CoinOpts> = {
...
+      hasMultiSend: true,
+      isUtxo: true,
+      isERCToken: false,
+      isStableCoin: false,
+      singleAddress: false
+    },
+    paymentInfo: {
+      paymentCode: 'BIP73',
+      protocolPrefix: { livenet: 'doge', testnet: 'doge' },
+      ratesApi: 'http://localhost:3232/bws/api/v3/fiatrates/doge',
```
Comment (Gamboster) [inline src/providers/currency/coin.ts:438]: localhost here

File: src/providers/incoming-data/incoming-data.ts [hunk 4]
```diff
@@ -1092,6 +1148,14 @@ export class IncomingDataProvider {
         title: this.translate.instant('XRP Address')
       };
 
+      // Plain Address (Dige)
```
Comment (Gamboster) [inline src/providers/incoming-data/incoming-data.ts:1151]: Dige -> Doge

File: src/providers/incoming-data/incoming-data.ts [hunk 5]
```diff
@@ -824,6 +870,11 @@ export class IncomingDataProvider {
       this.handleRippleUri(data, redirParams);
       return true;
 
+      // Dogec URI
```
Comment (Gamboster) [inline src/providers/incoming-data/incoming-data.ts:873]: Dogec -> Dogecoin / Doge

File: src/providers/derivation-path-helper/derivation-path-helper.ts [hunk 6]
```diff
@@ -15,6 +15,7 @@ export class DerivationPathHelperProvider {
     this.defaultBCH = "m/44'/145'/0'";
     this.defaultETH = "m/44'/60'/0'";
     this.defaultXRP = "m/44'/144'/0'";
+    this.defaultXRP = "m/44'/3'/0'";
```
Comment (Gamboster) [inline src/providers/derivation-path-helper/derivation-path-helper.ts:18]: defaultXRP  -> defaultDOGE

**Review by cmgustavo — APPROVED** [review]
🚀

**Review by cmgustavo — APPROVED** [review]
Huge work!

**Pull request merged**